dev-tools/mage/integtest.go

```diff
@@ -247,6 +248,10 @@ func initRunner(tester IntegrationTester, dir string, passInEnv map[string]strin
 		env["MAGEFILE_VERBOSE"] = "1"
 	}
 
+	if runtime.GOOS == "windows" {
+		env["TEST_COVERAGE"] = "false"
+	}
```

Comment:
Why disable coverage information for integration tests? I though system tests are the only problem. We might also want to test the architecture. AFIAK only 32bit windows is having problems, right?

Reply:
Every test seems to be impacted, not only system tests.

Reply:
> Every test seems to be impacted, not only system tests.

I see :/

Still we might want to disable TEST_COVERAGE only if the tests are run on a 32bit system.

Reply:
OK.

Reply:
Also, I will only disable it for Metricbeat....